src/PrestaShopBundle/Resources/config/services_dev/bundle/command.yml

```diff
@@ -63,3 +63,8 @@ services:
   PrestaShopBundle\Command\UpdateLicensesCommand:
     tags:
       - 'console.command'
+
+  PrestaShopBundle\Command\APIClientCommand:
+    autowire: true
+    tags:
+      - 'console.command'
```

Comment:
```suggestion
    autowire: true
    autoconfigure: true
```

Reply:
Use autoconfiguration to remove those ugly tags :D

Reply:
Good point, the other ones will be cleaned later I tried once to force autowiring and autoconfigue on the whole file but there were some errors I didn't have time to fix But ultimately this all should be simplified